docs/lua-api/classes/uenum.md

```diff
@@ -12,6 +12,25 @@
 
 ### ForEachName(LuaFunction Callback)
 
-- Iterates every `FName`/`Value` combination that belongs to this enum.
+- Iterates every `FName`, `Value` combination that belongs to this enum.
 - The callback has two params: `FName Name`, `integer Value`.
-- Return `true` in the callback to stop iterating.
\ No newline at end of file
+- Return `true` in the callback to stop iterating.
+
+### GetEnumNameByIndex(integer Index)
+- **Return type:** `FName`, `Integer`
```

Comment:
```suggestion
- **Return types:** `FName`, `Integer`
```
Grammatical error. I want it to be very clear that this is multiple return values.